java/app/src/main/java/com/fullstorydev/shoppedemo/ui/market/MarketFragment.java

```diff
@@ -21,13 +21,7 @@ public View onCreateView(@NonNull LayoutInflater inflater,
         marketViewModel =
```

Comment:
Similar to my comment on the Cart layout PR: this appears instantiated but unused.

Reply:
added this ViewModel instance serves as placeholder, since the actual VM logic is dependent on the data and networking classes to provide the data. 
Eventually VM will provide the LiveData for the Fragment - to be implemented

Reply:
had to remove the original boilerplate code because the TextView that it was looking for is deleted from the layout files.